SU-SWS/stanford_basic
# Move everything inside canvas-wrapper

# READY FOR REVIEW

# Summary
- Moves everything from html.html.twig that should be inside the page.html.twig template
- Moves the sticky behavior to the wrapper footer
- Removes the calls to the pattern functions in hopes to remove the dependency on ui_patterns

# Review By (Date)
- Dec 10th

# Urgency
- Low

# Steps to Test

1. Review code
2. Check out this branch and clear all caches
3. Change brand bar variant setting in appearance and clear caches to see change
4. Place a block in the footer and ensure it 'sticks' above the global footer/local footers

# Affected Projects or Products
- Local footer

# Associated Issues and/or People
- Local footer

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
